src/main/java/com/puppycrawl/tools/checkstyle/SuppressionsStringPrinter.java

```diff
@@ -0,0 +1,132 @@
+    private SuppressionsStringPrinter() {
+        // no code
+    }
+
+    /**
+     * Prints generated suppressions.
+     * @param file the file to process.
+     * @param suppressionLineColumnNumber line and column number of the suppression
+     * @param tabWidth length of the tab character
+     * @return generated suppressions.
+     * @throws IOException if the file could not be read.
+     * @throws CheckstyleException if the file is not a Java source.
+     */
+    public static String printSuppressions(File file, String suppressionLineColumnNumber,
+                                           int tabWidth) throws IOException, CheckstyleException {
+        if (VALID_SUPPRESSION_LINE_COLUMN_NUMBER_REGEX.matcher(suppressionLineColumnNumber)
+                .matches()) {
+            final FileText fileText = new FileText(file.getAbsoluteFile(),
+                    System.getProperty("file.encoding", StandardCharsets.UTF_8.name()));
+            final DetailAST detailAST = parseFileText(fileText);
+            final int lineNumber = extractLineNumber(suppressionLineColumnNumber);
+            final int columnNumber = extractColumnNumber(suppressionLineColumnNumber);
+            return generate(fileText, detailAST, lineNumber, columnNumber, tabWidth);
+        }
+        else {
+            final String exceptionMsg = String.format(Locale.ROOT,
+                    "%s does not match valid format 'line:column'.",
+                    suppressionLineColumnNumber);
+            throw new IllegalStateException(exceptionMsg);
+        }
+    }
+
+    /**
+     * Extracts line number from config property.
+     * @param suppressionLineColumnNumber line and column number of the suppression.
+     * @return line number.
+     */
+    private static int extractLineNumber(String suppressionLineColumnNumber) {
+        final int colonPosition = suppressionLineColumnNumber.indexOf(COLON_LITERAL);
+        final String lineNumber = suppressionLineColumnNumber.substring(0, colonPosition);
+        return Integer.parseInt(lineNumber);
+    }
+
+    /**
+     * Extracts column number from config property.
+     * @param suppressionLineColumnNumber line and column number of the suppression.
+     * @return column number.
+     */
+    private static int extractColumnNumber(String suppressionLineColumnNumber) {
+        final int colonPosition = suppressionLineColumnNumber.indexOf(COLON_LITERAL);
+        final String columnNumber = suppressionLineColumnNumber.substring(colonPosition + 1);
+        return Integer.parseInt(columnNumber);
+    }
+
+    /**
+     * Parses file and returns {@code DetailAST} object.
+     * @param text {@code FileText} object.
+     * @return {@code DetailAST} object.
+     * @throws CheckstyleException if the file is not a Java source.
+     */
+    private static DetailAST parseFileText(FileText text)
+            throws CheckstyleException {
+        return JavaParser.parseFileText(text, JavaParser.Options.WITH_COMMENTS);
```

Comment:
Do we really need a 1 line method wrap?
Why don't we inline this method?

Reply:
I see no reason too

Reply:
Done